deploy/uwsgi.yml

```diff
@@ -20,3 +20,8 @@ uwsgi:
   ignore-sigpipe: true
   ignore-write-errors: true
   disable-write-exception: true
+
+  logger-req: stdio
+  log-format: '"remote_addr": "%(addr)", "x_forwarded_for":"%(var.HTTP_X_FORWARDED_FOR)", "request_id":"%(var.HTTP_X_REQUEST_ID)", "remote_user":"%(user)", "bytes_sent":%(size), "request_time":%(secs), "status":%(status), "host":"%(host)", "request_proto":"%(proto)", "path":"%(uri)", "request_length":%(cl), "http_referer":"%(referer)", "http_user_agent":"%(uagent)"'
```

Comment:
Not sure if a plugin would be required for some json escaping. Some other project have decided to use a plugin. There's some details about the issue [here](https://www.velebit.ai/blog/tech-blog-uwsgi-json-logging/). E.g. [atv uses a plugin](https://github.com/City-of-Helsinki/atv/blob/main/.prod/uwsgi.ini#L2).

One thing to consider also would be if we would need the log to be in json format or not. :thinking:  

Seems that there's some issue related to this yaml-configuration. On my machine this produced the following type of log. Note that the `${msg}` part in the example below is actually a string which contains data which looks like a json.
```
{"time":"2024:07:31 13:19:40", "source":"uwsgi-req", '"remote_addr": "172.18.0.1", "x_forwarded_for":"-", "request_id":"-", "remote_user":"-", "bytes_sent":18671, "request_time":0.000206, "status":200, "host":"localhost:8083", "request_proto":"HTTP/1.1", "path":"/static/admin/css/responsive.css", "request_length":0, "http_referer":"http://localhost:8083/admin/login/?next=/admin/", "http_user_agent":"Mozilla/5.0 (X11; Linux x86_64; rv:128.0) Gecko/20100101 Firefox/128.0"'}
```

Tried locally to change the yaml formatted uwsgi configuration into the usual `ini` format and it was able to produce a more valid json.

Reply:
I think I had a discussion about this with some on in our team... 2 months ago. I recall ending up with removing the JSON escape thing, which I seem to have done here: https://github.com/City-of-Helsinki/kerrokantasi/commit/ffe3662bc369e2a3cf8224b5ef38f8eec2224a1d

I'm not quite sure why this would need to be in JSON format in the first place. It seems that other projects do this, but... why? Is it just another KuVa remnant or does it actually have a purpose?

Yeah, I had a lot of trouble with YAML. Most of my force-pushes in the draft stage were because of YAML. I'd prefer INI here, but, well, the configuration's in YAML. I suppose I could rewrite it in INI.